### PR TITLE
feat(FormStatus): Add Sbanken styles

### PR DESCRIPTION
**Pull request by Sundfjord**
Added Sbanken styles to match color scheme present in GlobalStatus/Section components.

### CODE REVIEW

File: packages/dnb-design-system-portal/src/docs/icons/form-status.mdx [hunk 1]
```diff
@@ -6,11 +6,19 @@ import {
 } from '@dnb/eufemia/src/components/form-status/FormStatus'
 
 ## Use the [**FormStatus**](/uilib/components/form-status) icons only
+<VisibilityByTheme hidden="sbanken">
```
Comment (tujoworker) [inline packages/dnb-design-system-portal/src/docs/icons/form-status.mdx:9]: Regarding the formatting. A newline between each new content type would be good. Also, when running prettier, it should remove two newlines to one.

File: packages/dnb-design-system-portal/src/docs/icons/form-status.mdx [hunk 2]
```diff
@@ -6,11 +6,19 @@ import {
...
+  - `InfoIcon`   <InfoIcon theme="sbanken" />
+  - `WarnIcon`   <WarnIcon />
+  - `ErrorIcon` <ErrorIcon />
+  - `MarketingIcon` <MarketingIcon />
+</VisibilityByTheme>
 
-- `InfoIcon`   <InfoIcon />
-- `WarnIcon`   <WarnIcon />
-- `ErrorIcon` <ErrorIcon />
-- `MarketingIcon` <MarketingIcon />
```
Comment (tujoworker) [inline packages/dnb-design-system-portal/src/docs/icons/form-status.mdx:22]: Here we get two newlines.

File: packages/dnb-eufemia/src/components/form-status/__tests__/__image_snapshots__/formstatus-for-sbanken-have-to-match-custom-content.snap.png
Comment (tujoworker) [inline packages/dnb-eufemia/src/components/form-status/__tests__/__image_snapshots__/formstatus-for-sbanken-have-to-match-custom-content.snap.png:1]: Should the icon be black?

File: packages/dnb-eufemia/src/components/form-status/__tests__/__image_snapshots__/formstatus-for-sbanken-have-to-match-icons-used-in-the-icon-component.snap.png
Comment (tujoworker) [inline packages/dnb-eufemia/src/components/form-status/__tests__/__image_snapshots__/formstatus-for-sbanken-have-to-match-icons-used-in-the-icon-component.snap.png:1]: Are they correct?

**Review by tujoworker — APPROVED** [review]
Nice work 👌 I just have some question and one formatting suggestion 🙏

File: packages/dnb-eufemia/src/components/form-status/__tests__/__image_snapshots__/formstatus-for-sbanken-have-to-match-custom-content.snap.png
Comment (Sundfjord) [reply packages/dnb-eufemia/src/components/form-status/__tests__/__image_snapshots__/formstatus-for-sbanken-have-to-match-custom-content.snap.png:1]: Yes, the info icon is (possibly temporarily) black in the Sbanken designs.

File: packages/dnb-eufemia/src/components/form-status/__tests__/__image_snapshots__/formstatus-for-sbanken-have-to-match-icons-used-in-the-icon-component.snap.png
Comment (Sundfjord) [reply packages/dnb-eufemia/src/components/form-status/__tests__/__image_snapshots__/formstatus-for-sbanken-have-to-match-icons-used-in-the-icon-component.snap.png:1]: No! Fixing.

**Pull request merged**